students/ThanhNhan/lesson09/test_mr_5.py

```diff
@@ -0,0 +1,177 @@
+import os
+import pytest
+from .mail5 import Donor, DonorCollection
+
+donor_db = {"Tom Cruise": [233.23, 12.17],
+            "Alyssa Nhan": [23., 23],
+            "Jessy Apple, IIS": [33.0],
+            "Jin Unknown": [12.0, 12.0]
+            }
+
+# Test object of the name
+def test_donor_name():
+    donor = Donor("Test ASdf")
+    assert donor.name == "Test ASdf"
+
+# Count how many time donation
+def test_add_donation():
+    donor = Donor("Fred Test")
+    donor.add_donations(500)
+    assert donor.num_donations == 1
+    donor.add_donations(320)
+    assert donor.num_donations == 2
+
+#Test dollar amount input
+def test_ck_char_input():
```

Comment:
nice!

but better to make these separate tests -- so if one fails, the others will still run.